manifests/backend-service.yml

```
@@ -0,0 +1,12 @@
+apiVersion: v1
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)